capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/photo/PhotoCropModifier.java

```diff
@@ -39,36 +43,61 @@ public void modify() {
                 Matrix matrix = new Matrix();
                 matrix.postRotate(rotation);
 
-                Bitmap originalBitmap = BitmapFactory.decodeByteArray(mPhoto.getData(),
+                Bitmap originalPhotoBitmap = BitmapFactory.decodeByteArray(mPhoto.getData(),
                         0, mPhoto.getData().length);
 
-                Bitmap rotatedBitmap = Bitmap.createBitmap(originalBitmap, 0, 0, originalBitmap.getWidth(),
-                        originalBitmap.getHeight(), matrix, false);
+                Bitmap rotatedPhotoBitmap = Bitmap.createBitmap(originalPhotoBitmap, 0, 0, originalPhotoBitmap.getWidth(),
+                        originalPhotoBitmap.getHeight(), matrix, false);
 
+                final float cropScalePercent = 0.2f;
 
-                int x1 = rotatedBitmap.getWidth() * aRect.left / mScreenSize[0];
-                int y1 = rotatedBitmap.getHeight() * aRect.top / mScreenSize[1];
-                int width1 = rotatedBitmap.getWidth() * aRect.width() / mScreenSize[0];
-                int height1 = rotatedBitmap.getHeight() * aRect.height() / mScreenSize[1];
+                // Scale the crop rect
+                final int scaledCropX = (int) (mCropRect.left - (mCropRect.width() * cropScalePercent));
+                final int scaledCropY = (int) (mCropRect.top - (mCropRect.height() * cropScalePercent));
+                final int scaledCropWidth = (int) (mCropRect.width() * (1 + cropScalePercent * 2));
+                final int scaledCropHeight = (int) (mCropRect.height() * (1 + cropScalePercent * 2));
 
```

Comment:
Awesome, just wondering why you have put the number `1` here `(1 + cropScalePercent * 2)` in this formula?

Reply:
The width and height increase by 40% (20% left side plus 20% right side, similarly for height with top and bottom side). So this means we multiply width and height by `1.4` if the scaling percentage is 20% (=`0.2`). To easily adjust the scaling percentage I broke that `1.4` up into `1 + scaling-percentage * 2`.